discharge_conditions_map/R/plot_utils.R

```diff
@@ -0,0 +1,68 @@
+plot_ww_map <- function(filename, us_states, col_sites){
+  png(filename = filename, width = 10, height = 6, res = 300, units = 'in')
+  
+  par(mai=c(0,0,0,0), omi=c(0,0,0,0), xaxs = 'i', yaxs = 'i')
+  
+  alpha_hex <- 'CC'
+  state_fill <- "#E0DED7" # play w/ this
+  shore_border <- "#7A8567"
+  plot(us_states, col = state_fill, border = 'white')
```

Comment:
looks like the float states won't have a border, but CONUS will?  Could add a CONUS boundary the same color as the background if you want the white lines only between states.

Reply:
yes, exactly. I did it this way so the islands of AK wouldn't be covered up by the border and those coasts will be sharper. Isn't the background the same color (white) already?

Reply:
ah forgot that the overall plot background is white 😁